data/src/main/java/com/android254/data/network/apis/OrganizersApi.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2022 DroidconKE
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.android254.data.network.apis
+
+import com.android254.data.network.Constants
+import com.android254.data.network.models.responses.OrganizersPagedResponse
+import com.android254.data.network.util.safeApiCall
+import io.ktor.client.*
+import io.ktor.client.call.*
+import io.ktor.client.request.*
+import javax.inject.Inject
+
+class OrganizersApi @Inject constructor(
+    private val client: HttpClient
+) {
+
+    suspend fun fetchOrganizers(perPage: Int, page: Int): OrganizersPagedResponse =
+        safeApiCall {
+            client.get("${Constants.BASE_URL}/organizers?per_page=$perPage&page$page").body()
```

Comment:
@MamboBryan I think you need to make use of this API. - `/organizers/{{org_slug}}/team` with filter `?type=individual/company`

Reply:
droidconKe in this context is the organizer, what you need I think is the organizing team

Reply:
The one I left in the comment ealier